subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/tasks/JacocoReportBase.java

```diff
@@ -109,6 +110,7 @@ public ConfigurableFileCollection getSourceDirectories() {
     /**
      * Source sets that coverage should be reported for.
      */
+    @Classpath
     @PathSensitive(PathSensitivity.RELATIVE)
     @InputFiles
```

Comment:
You can remove both `@InputFiles` and `@PathSensitive`.

Reply:
Good point. I was looking at the docs of `@Classpath` and it said for backwards compatibility to keep them, but we aren't plugin authors. This is Gradle itself. :)

Reply:
Done.